### ISSUE TITLE
Add gradient support to the notice partial

### ISSUE THREAD

**Issue by cswimr** [Galaxypedia-Wiki/Galaxypedia#42]
Some of the old notice templates on the Mediawiki site had gradients for border colors. We can replicate this with Tailwind CSS, but it'll be a bit more complicated.

**Comment by smallketchup82** [comment]
fwiw i’m against this change. i’ve never seen the need for gradients on notices on the galaxypedia and only kept them for historical reasons.

this is more of a personal preference than anything else. color coded notices make it easy for people to quickly identify their contents without having to reread them. gradients, on the other hand, require a bit more effort to decipher (take the prototype x-1 notice as an example).

i’m open to debate, but i need to be convinced that gradients offer something beyond their aesthetic appeal.

**Comment by cswimr** [comment]
> color coded notices make it easy for people to quickly identify their contents without having to reread them. gradients, on the other hand, require a bit more effort to decipher (take the prototype x-1 notice as an example).

I disagree. If anything, gradients make it easier to identify the content of a notice without reading it. If it's your first time reading a notice, you won't connect a color to it yet anyway. And, as more notices are added, some notices will have the same (or similar) colors as other notices, and that connection you make between the color and the notice will be meaningless as you now need to read the notice anyway to see which notice is which. Gradients remove this problem, because you make a connection between the gradient as a whole instead of the single solid color. I don't see how a gradient removes that benefit.

**Comment by shooter7620** [comment]
Not to bump this for no reason, but I found gradients useful for some of the newer notice templates as the band of colors available that weren't already used by another template (or at least fairly similar to another color) ended up becoming too slim.

While I can understand the emphasis on actually reading the content of a notice, color coding with unique colors/gradients is a nice way to quickly identity what type of notice is visible at a glance, as cswimr said.

If gradients are too difficult to incorporate, then why not assign colors to categories of notices instead of each notice having their own unique color? It doesn't really seem to make sense to kind of defeat the point of having the unique color coding we have had until now by getting rid of gradients.

Alternatively, maybe the templates can be modified to have a different style for the border, allowing for separate colors on two different sections of the border?
